app/components/DR/ActionCards/ActionCards.js

```diff
@@ -8,45 +8,77 @@ import {
 import Icon from 'react-native-vector-icons/FontAwesome5';
 
 import Colors from '../../../constants/colors';
+import { GetStoreData } from '../../../helpers/General';
 import languages from '../../../locales/languages';
 import styles from './styles';
 
 const { ORANGE, GREEN, BLUE_RIBBON } = Colors;
 
-export function Feels({ navigation }) {
-  return (
-    <View>
-      <Card style={styles.bigCards}>
-        <View style={styles.auroraContainer}>
-          <Icon name='heartbeat' color={ORANGE} size={wp('7%')} />
-          <Text style={[styles.textHeader, { marginLeft: 8 }]}>
-            {languages.t('label.report_symptoms_title')}
-          </Text>
-        </View>
-        <View style={{ flexDirection: 'column' }}>
-          <Text style={[styles.text, { width: '90%' }]}>
-            {languages.t('label.report_symptoms_description')}
-          </Text>
-          <View style={{ justifyContent: 'center' }}>
-            <Button
-              onPress={() => navigation.navigate('ReportScreen')}
-              style={[
-                styles.buttons,
-                {
-                  backgroundColor: GREEN,
-                  width: wp('70%'),
-                  marginTop: 15,
-                },
-              ]}>
-              <Text style={styles.buttonText}>
-                {languages.t('label.report_symptoms_label')}
-              </Text>
-            </Button>
+export class Feels extends Component {
```

Comment:
We could simplify thing by using hooks instead of class 

```js
import React, { useState, useEffect } from 'react';
...

export function Feels({ navigation }) {
  const [isInfected, setIfInfected] = setState(false);

  useEffect(() => {
   GetStoreData('positive', false).then((positive) => setIfInfected(positive));
  }, []);
...
```